### DISCUSSION TITLE
Is it possible to get list of upcoming IPOs?

### DISCUSSION THREAD

**Discussion by gety9** [joqueka/bf4py#18] [Q&A]
Is it possible to get list of upcoming IPOs using bf4py?
I see

```
bf4py.company
.ipo_details(...)

```

and 

```
bf4py.news
.news_by_category(...)
```

but not sure if it's it, so decided to ask :)

If not, maybe you could suggest where i can get such data? (i am new to boerse-frankfurt/xetra)

**Comment by joqueka** [comment]
Hi

I am not aware of any way to retrieve upcoming IPOs via an API endpoint. There is only one subpage that you can evaluate manually:
https://www.boerse-frankfurt.de/aktien/neuemissionen

bf4py does not implement a parser, so access to it is not provided.
